finality-aleph/src/party/member.rs

```diff
@@ -44,3 +59,23 @@ pub fn task<
     let handle = spawn_handle.spawn_essential("aleph/consensus_session_member", task);
     Task::new(handle, stop)
 }
+
+fn rotate_saved_unit_files(stash_path: &Path, session_id: u32) -> Result<(File, File), io::Error> {
```

Comment:
I have some problems with catching the intention behind `index` variable here (is it supposed to indicate different runs or restarts or ...?), so maybe some explanation in docstring would help